onnxruntime/core/mlas/lib/pooling.cpp

```diff
@@ -1304,11 +1304,12 @@ Return Value:
 #endif
 }
 
+template<typename T8Bits>
```

Comment:
template defined in cpp file can be linked?

Reply:
right, given the explicit instantiations below